### PR TITLE
Use auto-generated flyway properties table in documentation

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: docs/src/main/asciidoc/flyway.adoc [hunk 1]
```diff
@@ -49,74 +49,8 @@ First, you need to add the datasource config to the `{config-file}` file
...
-**default:** false
-
-`quarkus.flyway.baseline-version`::
-The version to tag an existing schema with when executing baseline +
-**default:** 1
-
-`quarkus.flyway.baseline-description`::
-The description to tag an existing schema with when executing baseline +
-**default:** '<< Flyway Baseline >>'
+include::{generated-dir}/config/quarkus-flyway.adoc[opts=optional, leveloffset=+1]
```
Comment (gsmet) [inline docs/src/main/asciidoc/flyway.adoc]: Ah, cool, I wanted to do that forever... Did you check that all the information there are in the javadoc?

**Review by gsmet — DISMISSED** [review]
Asking a question. Please dismiss if OK :).

File: docs/src/main/asciidoc/flyway.adoc (hunk 1 above)
Comment (gastaldi) [reply docs/src/main/asciidoc/flyway.adoc]: Yes

**Review by gsmet — APPROVED** [review]
Cool, then!

**Pull request merged**